app/src/main/java/com/github/geohunt/app/model/database/api/Challenge.kt

```diff
@@ -45,11 +45,19 @@ interface Challenge
      * but the author
      */
     val correctLocation: Location
-    
+
     /**
      * A list of reference to all of the claims from users
      */
     val claims: List<LazyRef<Claim>>
 
-}
+    /**
+     * A list of references to all of the likes from users
+     */
+    val likes: List<LazyRef<User>>
 
+    /**
+     * The number of likes for this challenge
+     */
+    var nbLikes: Int
```

Comment:
```suggestion
    val numberOfLikes : Int
```

Try to avoid using abbreviation (even though I am sure I am going to do the same mistake in my next PR 😅 )

Reply:
I have refactored the name of the variable to numberOfLikes